riff-cli/pkg/osutils/osutils.go

```diff
@@ -88,27 +87,6 @@ func Path(filename string) string {
 	return filepath.Join(strings.Split(path, "/")...)
 }
 
-func ExecWaitAndStreamOutput(cmdName string, cmdArgs []string) {
```

Comment:
This should be a common function for logs as well. That's what I did in the recently closed PR. But we can refactor later